src/com/seafile/seadroid2/cameraupload/CameraUploadService.java

```diff
@@ -267,6 +267,18 @@ protected File doInBackground(Void... params) {
 
         @Override
         protected void onPostExecute(File photo) {
```

Comment:
Why not check the network status [here](https://github.com/haiwen/seadroid/tree/ffe91ce4874735bc9c4b36d3c92689767561b47c/src/com/seafile/seadroid2/cameraupload/CameraUploadService.java#L188), so that we would not create this asynctask at all.